manager/system_handler.py

```diff
@@ -317,6 +327,7 @@ def handle_get(cls, **kwargs):
                 record["attributes"] = system
                 record["id"] = system["inventory_id"]
                 record["type"] = "system"
+                # record["inventory_group"] = system.get("inventory_group", [])
```

Comment:
Is this correct?

Reply:
Even without this line, the data contain inv group info, so I'll remove it